### ISSUE TITLE
Map nan or 0

### ISSUE THREAD

**Issue by Ypetrol** [QinLab-WFU/DSPH#4]
Could you tell me the learning rate used for training the NUS-WIDE dataset in the paper “Deep multi-similarity hashing via label-guided network for cross-modal retrieval”? Thank you very much.

**Comment by QinLab-WFU** [comment]
你好, NUS-WIDE数据集的学习率设为0.001.
